packages/experiments/src/components/Card/CardItem/CardItem.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { getGlobalClassNames } from '../../../Styling';
+import { ICardItemComponent, ICardItemStylesReturnType } from './CardItem.types';
+
+const GlobalClassNames = {
+  root: 'ms-CardItem'
+};
+
+export const styles: ICardItemComponent['styles'] = (props, theme): ICardItemStylesReturnType => {
+  const { preventPadding } = props;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  return {
+    root: [
+      theme.fonts.medium,
+      classNames.root,
+      {
+        width: 'auto',
+        height: 'auto'
+      },
+      preventPadding && {
+        marginLeft: '-12px',
+        marginRight: '-13px'
```

Comment:
Is there rationale for these values and why they differ?

Reply:
`12px` is the default padding on the `Card`. The `marginRight` adds one extra pixel to compensate for the `boxShadow` on the right being `1px` wide.

Reply:
I chatted with the designers this morning and they'll be sending along designs soon. The final version will have a symmetrical shadow so we can adjust this when it comes in.

Reply:
Hmm, I wonder if we should be linking these values. Like make a const inside this function:
```tsx
const defaultPadding = '12px';
```
And then using `defaultPadding` in styling vs. hardcoded values. I know that's not typically our standard but it might help prevent regressions over time.

Reply:
Actually the '12px` is scattered across two styles files, which is even more likely to regress if Card's value changes yet Item's does not. There are a couple of ways we could do this.. if not by linking a shared const value, then perhaps some kind of unit test.

Reply:
After trying for almost a full-day to get a way to compare the `padding` and `margin` of `Card` and `CardItem` respectively via a unit testing approach I've found that its pretty much impossible via jest/enzyme.

Given that, I finally chose to use a `defaultPadding` const to link both. I've added a couple of snapshots for it.

Not feeling very confident about this change though, so it would be great if I could get some feedback on it.

Reply:
How about just setting up a unit test on the styles functions?

Reply:
Leveraged the styles functions to write a unit test. I would still like feedback on this approach though, as I'm not 100% confident on the way I did it.

Reply:
It is more complicated than I was hoping. Per our discussion, I think it's worth exploring this just a bit more as a future example with possible use of `concatStyleSets` to simplify the test.

Reply:
Per our discussion it seems that there is no utility that completely fits our needs here to simplify the test. We'll go with our approach here for now, but maybe it would be good to have such a utility in the future.